### PR TITLE
webhook: improve node affinity performance for MultiQuotaTree

### PR DESCRIPTION
**Pull request by shaloulcy**
### Ⅰ. Describe what this PR does

webhook: improve node affinity performance for MultiQuotaTree

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/webhook/elasticquota/quota_handler.go [hunk 1]
```diff
@@ -71,6 +79,18 @@ func (qt *quotaTopology) OnQuotaUpdate(oldObj, newObj interface{}) {
 		delete(qt.quotaHierarchyInfo[oldQuotaInfo.ParentName], oldQuotaInfo.Name)
 		qt.quotaHierarchyInfo[newQuotaInfo.ParentName][newQuotaInfo.Name] = struct{}{}
 	}
+
+	oldNamespaces := extension.GetAnnotationQuotaNamespaces(oldQuota)
+	newNamespaces := extension.GetAnnotationQuotaNamespaces(newQuota)
+	if reflect.DeepEqual(oldNamespaces, newNamespaces) {
```
Comment (hormes) [inline pkg/webhook/elasticquota/quota_handler.go:85]: if !reflect.DeepEqual(oldNamespaces, newNamespaces) ?

Comment (shaloulcy) [reply pkg/webhook/elasticquota/quota_handler.go]: done

Comment (shaloulcy) [reply pkg/webhook/elasticquota/quota_handler.go]: done

**Review by eahydra — COMMENTED** [review]
/lgtm

**Pull request merged**